src/data/polynomial/coeff.lean

```diff
@@ -60,7 +60,7 @@ have hite : ∀ a : ℕ × ℕ, ite (a.1 + a.2 = n) (coeff p (a.fst) * coeff q (
   (λ h, absurd (eq.refl (0 : R)) (by rwa if_neg h at ha)),
 calc coeff (p * q) n = ∑ a in p.support, ∑ b in q.support,
     ite (a + b = n) (coeff p a * coeff q b) 0 :
-  by { simp only [mul_def, coeff_sum, coeff_single], refl }
+  by { simp only [add_monoid_algebra.mul_def, coeff_sum, coeff_single], refl }
```

Comment:
`open set (hiding mul_def)` in this file?

Reply:
I considered that, but decided against it. Doing that is a temporary patch, and not a real fix. `mul_def` is too ambiguous to be used without prefix.